app/konnect/api-products/productize-service.md

```diff
@@ -47,7 +47,7 @@ The response body will include an `id` field, denoting the unique identifier for
 {% navtab Konnect UI %}
 After creating a new API product, you can attach an API product version to it.
 
-1. From the API product builder, select **Product Versions**, then select **New Version**.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), click the API product you want to create the version for and then click **Product Versions**, then click **New Version**.
 
 1. Enter a version name. For example `v1`.
      A version name can be any string containing letters, numbers, or characters;
```

Comment:
change "A service can have multiple versions." to "An API Product can have multiple versions."